### PR TITLE
[onert] Move random input generation into randomgen

### PR DESCRIPTION
**Pull request by glistening**
It moves random input generation code into randomgen.*.
Also, it cleans up trivials - put header in order, removing blank lines.

ONE-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

File: tests/tools/nnpackage_run/src/h5formatter.h [hunk 1]
```diff
@@ -29,12 +29,12 @@ namespace nnpkg_run
...
 {
 public:
-  H5Formatter(nnfw_session *sess) : session(sess) {}
+  H5Formatter(nnfw_session *sess) : session_(sess) {}
   void loadInputs(const std::string &filename, std::vector<Allocation> &inputs);
   void dumpOutputs(const std::string &filename, std::vector<Allocation> &outputs);
 
 private:
-  nnfw_session *session;
+  nnfw_session *session_;
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/h5formatter.h:37]: I will create other PR for renaming `session` to `session_` if reviewer prefers.

File: tests/tools/nnpackage_run/src/h5formatter.cc [hunk 2]
```diff
@@ -18,10 +18,8 @@
 #include "nnfw.h"
 #include "nnfw_util.h"
 
-#include <cassert>
 #include <iostream>
 #include <stdexcept>
-#include <cstdlib>
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/h5formatter.cc:24]: Removing unnecessary headers. It could be in other PR if reviewer prefers

File: tests/tools/nnpackage_run/src/nnpackage_run.cc [hunk 3]
```diff
@@ -32,22 +32,11 @@
 #include <chrono>
 #include <cstdlib>
 #include <iostream>
+#include <libgen.h>
```
Comment (glistening) [inline tests/tools/nnpackage_run/src/nnpackage_run.cc:35]: Putting headers in order could be another PR if reviewer prefers

**Comment by YongseopKim** [comment]
The title would be better if [onert] -> [nnpkg_run].

File: tests/tools/nnpackage_run/src/randomgen.cc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+template <class T> void randomData(nnfw::misc::RandomGenerator &randgen, void *data, uint64_t size)
+{
+  for (uint64_t i = 0; i < size; i++)
+    reinterpret_cast<T *>(data)[i] = randgen.generate<T>();
+}
+
+void RandomGenerator::generate(std::vector<Allocation> &inputs)
+{
+  // generate random data
+  const int seed = 1;
```
Comment (YongseopKim) [inline tests/tools/nnpackage_run/src/randomgen.cc:36]: How about changing `seed` not static but dynamic? I mean such as time()...

Comment (glistening) [reply tests/tools/nnpackage_run/src/randomgen.cc:36]: @YongseopKim Thank you for comment.

First, it is beyond this PR.

Second, there has been discussion about constant seed vs. changing seed. I am for const since it gives deterministic result. If we use different seed for each CI run, it is hard to know the regression comes from different seed or PR itself.

If someone or we would like to strengthen the test by testing more input values (it is good), we may add additional argument like #2622, and run several times with different seed daily between midnight to dawn when CI is likely to be idle.
